shared/redux/rtk-apis/users/users.api.ts

```diff
@@ -1,13 +1,10 @@
-import { TApiResponse } from "@/shared/typedefs";
-
 import projectApi from "../api.config";
 import { TTokenizedUser } from "../auth/auth.types";
 
 const usersApi = projectApi.injectEndpoints({
   endpoints: (builder) => ({
     me: builder.query<TTokenizedUser, void>({
-      query: () => "users/me",
-      transformResponse: (response: TApiResponse<TTokenizedUser>) => response.data,
+      query: () => "auth/me",
```

Comment:
Any reason why we're moving this from "users" to "auth"?

Reply:
@akib-raihan-sazim If a question is asked in any of the PRs, please reply even if the the conversation was done offline. This helps other reviewers.